content/en/data_jobs/databricks.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Enable Data Jobs Monitoring for Databricks
+kind: documentation
+further_reading:
+    - link: '/data_jobs'
+      tag: 'Documentation'
+      text: 'Data Jobs Monitoring'
+---
+
+{{< callout url="https://forms.gle/PZUoEgtBsH6qM62MA" >}}
+Data Jobs Monitoring is in private beta. Fill out this form to join the wait list.
+{{< /callout >}} 
+
+[Data Jobs Monitoring][7] gives visibility into the performance and reliability of your Apache Spark and Databricks jobs.
+
+## Setup
+
+Follow these steps to enable Data Jobs Monitoring for Databricks.
+
+1. [Configure the Datadog-Databricks integration](#configure-the-datadog-databricks-integration) with your Databricks API token.
+1. [Install the Datadog Agent](#install-the-datadog-agent-on-your-databricks-clusters) on your Databricks cluster(s).
+1. [Add your Datadog API key](#add-your-datadog-api-key-in-databricks) in Databricks.
+
+### Configure the Datadog-Databricks integration 
+
+1. In your Databricks workspace, click **User Settings**. Then click on **Access Tokens**.
+1. Click **Generate new token**, enter a comment, and click **Generate**. Take note of your token.
+1. In Datadog, open the Databricks integration tile.
+1. On the **Configure** tab, click **Add New**. 
+1. Enter a workspace name, your Databricks workspace URL, and the Databricks token you generated.
+   {{< img src="data_jobs/databricks/configure-token.png" alt="In the Datadog-Databricks integration tile, a Databricks workspace is displayed. This workspace has a name, URL, and API token." style="width:100%;" >}}
+
+
+### Install the Datadog Agent on your Databricks cluster(s)
+
+Use [this init script][2] to install the Datadog Agent on a specific Databricks cluster, or globally.
+
+{{< tabs >}}
+{{% tab "On a specific cluster" %}}
+1. Download the [init script][1].
+1. In Databricks, on the cluster configuration page, click the **Advanced options** toggle.
+1. At the bottom of the page, go to the **Init Scripts** tab.
+   {{< img src="data_jobs/databricks/init_scripts.png" alt="Databricks UI, cluster configuration advanced options,  Init Scripts tab. A 'Destination' drop-down and an 'Init script path' file selector." style="width:80%;" >}}
+   - Under the **Destination** drop-down, select `Workspace`.
+   - Under **Init script path**, enter the path to your init script.
+   - Click **Add**.
+
+[1]: /resources/sh/data_jobs/datadog_databricks_job_monitoring_init_without_logs.sh
+{{% /tab %}}
+{{% tab "Global" %}}
+1. In Databricks, go to **Admin Settings** and click the **Global Init Scripts** tab.
```

Comment:
This Databricks UI also recently changed

The path is now: Admin Settings -> Compute -> Manage Global init scripts

<img width="1291" alt="image" src="https://github.com/DataDog/documentation/assets/99249445/7a5a826f-9248-47db-aaab-456489ca80d7">